### PR TITLE
Add software “solidtime”

### PR DESCRIPTION
**Pull request by korridor**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [X] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [X] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [X] Comments and unused optional fields have been removed.
- [X] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [X] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [X] Any software project you are adding to the list is actively maintained.
- [X] Any software project you are adding was first released more than 4 months ago.
- [X] Any software project you are adding has working installation instructions.
- [X] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Comment by Rabenherz112** [comment]
Hi, thanks for your contribution.

Currently, this project has a release, but it is not yet 4 months old. Our guidelines require that _Any software project you are adding was first released more than 4 months ago_. This count initiates only after a release has been created to ensure users need not rely on the latest development version to use the project.

I'll go ahead and close it for now to keep the PR section focused on active tasks. Once the first release is four months old, feel free to resubmit it to awesome-selfhosted, or you can create an issue instead (we don't close issues; we just tag them to indicate they need to mature).

Thanks for understanding, and good luck with this project.

**Comment by korridor** [comment]
@Rabenherz112 
Thanks for the feedback! I wanted to inform you that the first release of solidtime is now older than 4 months.
Would be great if you could look at the PR again. I'll also create an issue and link this PR.

File: software/solidtime.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: "solidtime"
+website_url: "https://www.solidtime.io"
+source_code_url: "https://github.com/solidtime-io/solidtime"
+description: "solidtime is a modern time tracking application for freelancers and agencies."
+licenses:
+  - AGPL-3.0
+platforms:
+  - PHP
```
Comment (Ki-er) [inline software/solidtime.yml:8]: >https://docs.solidtime.io/self-hosting/intro

PHP can be removed

```suggestion
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
> https://docs.solidtime.io/self-hosting/intro

I would improve the docs a little too. Include a sample docker command to install for example, add some detail etc

**Comment by korridor** [comment]
@Ki-er I'll remove "PHP" from the platforms, but I don't really understand your comment regarding the docs.

Do you mean the docs should include more information or do you mean I should link the docs somehow?

**Comment by Ki-er** [comment]
> Do you mean the docs should include more information or do you mean I should link the docs somehow?

Docs should include more information, such as how to use docker to install it, i.e a sample command to get it up and running

**Comment by korridor** [comment]
@Ki-er The link you sent is only the introduction there are many more pages (see navigation on the left) and a whole GitHub repository with self-hosting examples.
Link to the docker self-hosting guide: https://docs.solidtime.io/self-hosting/guides/docker
Link to the self-hosting examples repository (also linked in the docs): https://github.com/solidtime-io/self-hosting-examples

File: software/solidtime.yml (hunk 1 above)
Comment (korridor) [reply software/solidtime.yml]: Just removed that in the PR.

**Comment by korridor** [comment]
@Ki-er I just updated the docs page to link more clearly to docker self-hosting guide: https://docs.solidtime.io/self-hosting/intro

File: software/solidtime.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: "solidtime"
+website_url: "https://www.solidtime.io"
+source_code_url: "https://github.com/solidtime-io/solidtime"
+description: "solidtime is a modern time tracking application for freelancers and agencies."
```
Comment (nodiscc) [inline software/solidtime.yml:4]:
```suggestion
description: "Modern time tracking application for freelancers and agencies."
```

https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**